app/controllers/api/v1/admin/trends/statuses_controller.rb

```diff
@@ -1,7 +1,38 @@
 # frozen_string_literal: true
 
 class Api::V1::Admin::Trends::StatusesController < Api::V1::Trends::StatusesController
-  before_action -> { authorize_if_got_token! :'admin:read' }
+  include Authorization
+
+  before_action -> { authorize_if_got_token! :'admin:read' }, only: :index
+  before_action -> { authorize_if_got_token! :'admin:write' }, except: :index
+
+  after_action :verify_authorized
+
+  def index
+    authorize [:admin, :status], :index?
```

Comment:
Same remark as with `Api::V1::Admin::Trends::LinksController` with regards to changing the endpoint's behavior.

Reply:
You're right. I got a little overzealous writing the tests for these new methods. I have fixed it.